framework/src/bundle/BundleRegistry.cpp

```diff
@@ -140,6 +140,12 @@ namespace cppmicroservices
 
     std::vector<Bundle>
     BundleRegistry::Install(std::string const& location, BundlePrivate*, cppmicroservices::AnyMap const& bundleManifest)
+    {
+        return Install1(location, bundleManifest, nullptr);
```

Comment:
I'm not sure why there is this new sub call. 

Purely speaking as someone who frequents the install and start and stop calls across this code base, the presence of start, start0, start1... causes confusion when trying to understand code flow. So if we need this call, I would prefer it called something else

Reply:
Ended up renaming it to `BundleRegistry::InstallWithContainer`, is that ok?

The basic issue was that I need to pass in a mocked `BundleResourceContainer`, but the existing `Install` constructed it within the function.  I tried to use default arguments so as to avoid adding a whole new function, but something about setting the container to `nullptr` by default caused a segfault in the existing tests.  Not ideal I realize, but in order to mock bundles it was absolutely necessary to mock `BundleResourceContainer` somehow.